libsol/stake_instruction.c

```diff
@@ -95,6 +99,27 @@ static int parse_stake_initialize_instruction(
     return 0;
 }
 
+static int parse_stake_initialize_checked_instruction(
+    Parser* parser,
+    const Instruction* instruction,
+    const MessageHeader* header,
+    StakeInitializeInfo* info
+) {
+    InstructionAccountsIterator it;
+    instruction_accounts_iterator_init(&it, header, instruction);
+
+    BAIL_IF(instruction_accounts_iterator_next(&it, &info->account));
+    // Skip rent sysvar
+    BAIL_IF(instruction_accounts_iterator_next(&it, NULL));
+    BAIL_IF(instruction_accounts_iterator_next(&it, &info->stake_authority));
+    BAIL_IF(instruction_accounts_iterator_next(&it, &info->withdraw_authority));
+
+    // No lockup on checked instructions
+    info->lockup. present = StakeLockupHasNone;
```

Comment:
I was going to suggest that these stray spaces be removed throughout, but I see now that I introduced them :sweat_smile: 

Must've got carried away with a find/replace at some point :thinking:

Reply:
Haha, I thought it was a stylistic choice ;) There are just a handful of them, so I'll clean up